pkg/agent/controller/egress/egress_controller.go

```diff
@@ -65,6 +66,12 @@ const (
 	// maxEgressMark is the maximum mark of Egress IPs can be configured on a Node.
 	maxEgressMark = 255
 
+	// minEgressMeter is the minimum meter id of Egress Meters can be configured on a Node.
+	// Since meter already has been used in Antrea other features, so reserve 1-10 for Antrea other features.
```

Comment:
Antrea other features -> other Antrea features

Reply:
Thanks for the review, I will update it.